com.github.rajsolai.textsnatcher.yml

```
@@ -0,0 +1,27 @@
+app-id: com.github.rajsolai.textsnatcher
+
+runtime: org.gnome.Platform
+runtime-version: "40"
+base: io.elementary.BaseApp
+base-version: "juno-20.08"
+sdk: org.gnome.Sdk
+command: com.github.rajsolai.textsnatcher
+
+finish-args:
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--share=network"
+  - "--device=dri"
+  - "--filesystem=/home"
```

```suggestion
  - "--filesystem=/home"
```

1. this won't work
2. you should set it to `xdg-pictures`

---

Sure I will set it to pictures directory :+1: